tools/src/code-review/reviewers/reviewForbiddenFiles.ts

```diff
@@ -0,0 +1,54 @@
+import minimatch from 'minimatch';
+import path from 'path';
+import prettyBytes from 'pretty-bytes';
+
+import Git from '../../Git';
+import { ReviewInput, ReviewOutput, ReviewStatus } from '../types';
+
+const FILE_SIZE_LIMIT = 5 * 1000 * 1000; // 5MB
+const PRETTY_FILE_SIZE_LIMIT = prettyBytes(FILE_SIZE_LIMIT);
+
+const IGNORED_PATHS = ['android/versioned-abis/**/*.aar'];
+
+export default async function ({ pullRequest, diff }: ReviewInput): Promise<ReviewOutput | null> {
+  const listTree = await Git.listTreeAsync(
+    pullRequest.head.sha,
+    diff.filter((file) => !file.deleted).map((file) => file.path)
+  );
+
+  const fileReports = listTree
+    .map((file) => {
+      const logs: string[] = [];
+
+      // We may ignore some paths where the files are allowed to be bigger (e.g. versioned .aar files).
+      if (IGNORED_PATHS.some((pattern) => minimatch(file.path, pattern))) {
+        return null;
+      }
+      if (path.extname(file.path).substr(1) === 'gif') {
+        logs.push(`**GIF** format is forbidden, please consider using **MP4** format`);
```

Comment:
```suggestion
        logs.push(`**GIF** files are forbidden, please consider using **MP4** instead`);
```

also, not super important here but i imagine in the future we'll want to add some other disallowed files.. maybe .DS_Store for example? haha. could be worth pulling out into a deny list like we have done with `IGNORED_PATHS`

Reply:
Yeah, I started from having a list of formats, but I also want to emphasize to use MP4 instead of GIF, for DS_Store the message would be different, so it probably doesn't make sense to build up additional logic right now when we have just one format 😉